core/common/src/thrift/block_worker.thrift

```diff
@@ -113,5 +121,5 @@ service BlockWorkerClientService extends common.AlluxioService {
     /** the id of the block being accessed */ 1: i64 blockId,
     /** the id of the current session */ 2: i64 sessionId,
     )
-    throws (1: exception.AlluxioTException e)
+    throws (1: exception.AlluxioTException e, 2: exception.ThriftIOException ioe)
```

Comment:
Similar as the argument mentioned in another comment (Jiri, Bin and Pei discussed), this is backward compatible.

old client -> new server: the old client won't trigger this new exception because it still goes to FileSystemWorker to read UFS block.

new client -> old server. not considered. it won't work.